### ISSUE TITLE
No error details when CompileAssetCatalog or CompileXIB failed

### ISSUE THREAD

**Issue by truebit** [supermarin/xcpretty#196]
No error details when CompileAssetCatalog or CompileXIB failed
it only output something like:

```
The following build commands failed:
        CompileXIB test/classes/home/home/home.xib
(1 failure)
```

or 

```
The following build commands failed:
    CompileAssetCatalog build/Build/Intermediates/ArchiveIntermediates/test/IntermediateBuildFilesPath/UninstalledProducts/test.appex test/images.xcassets
(1 failure)
```

**Comment by zhouzhongyuan** [comment]
## Have you solved this problem？

**Comment by zhouzhongyuan** [comment]
I had the same error.
I solved by change the app icon to 120*120

**Comment by sureshxaton** [comment]
My app icon is already 120x120.. Still the build fails
The following build commands failed:
	CompileAssetCatalog <https://URL_MYAPPNAME.app> AppName/Supporting\ Files/Assets.xcassets

**Comment by supermarin** [comment]
Sorry about that, the fix is already in but shipping in 0.3
